AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Tenant/Index.cshtml

```diff
@@ -24,7 +24,15 @@
             @foreach (var item in Model) {
                 <tr>
                     <td>
+                        @if (String.IsNullOrEmpty(item.LogoUrl))
```

Comment:
Indeed.  I find it's sometimes clearer to write it out as I did it there, though, and it avoids potential problems with spurious spaces being appended to the start/end of the text inside the A element (after, say, the editor does some automatic indenting).  (Razor's shorthand `@( test ? "opt 1" : "opt 2")` isn't usable here because one of the options is an HTML tag).

More than happy to change it though; I'd also be interested to learn if there are any established best practices on this topic.  I've never come across any.